invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDaoHelper.java

```diff
@@ -227,7 +228,13 @@ public InvoiceItemModelDao createAdjustmentItem(final EntitySqlDaoWrapperFactory
     }
 
     public void populateChildren(final InvoiceModelDao invoice, final List<Tag> invoicesTags, final EntitySqlDaoWrapperFactory entitySqlDaoWrapperFactory, final InternalTenantContext context) {
```

Comment:
Do we have HA tests that call this implicitly? Since we have to duplicate the logic, and the HA one is a bit subtle, would be good to make sure we didn't introduce any regression.

Reply:
This code is pulled anytime there is an invoice generation or a payment associated with an invoice -- just to name those 2 use cases -- so will definitely be part of our test suite for HA. There could be bugs, that we don't see and that are not covered by our tests, but the same could be said of our existing logic.

Reply:
@pierre Update the PR with additional tests. See 5a10a5fb6027fc5cb